src/sentry/api/serializers/models/dashboard.py

```diff
@@ -143,11 +143,26 @@ def get_attrs(self, item_list, user):
         return result
 
     def serialize(self, obj, attrs, user, **kwargs):
+        page_filter_keys = ["environment", "range", "start", "end"]
+        dashboard_filter_keys = ["releases"]
         data = {
             "id": str(obj.id),
             "title": obj.title,
             "dateCreated": obj.date_added,
             "createdBy": serialize(obj.created_by, serializer=UserSerializer()),
             "widgets": attrs["widgets"],
+            "projects": [project.id for project in obj.projects.all()],
```

Comment:
If you ever render a list of dashboards, this will emit N queries. You could pre-fetch the dashboard : project listings in `get_attrs()`

Reply:
This is currently only implemented on the detail serializer, so this information isn't queried for in the list view. I don't think I need to pre-fetch here?

Reply:
If there is only one dashboard ever getting a list or projects what you have will be fine.